suite/tests/client-interface/inline.dll.c

```diff
@@ -207,7 +207,7 @@ codegen_instrumentation_funcs(void)
         dr_log(dc, LOG_EMIT, 3, "Generated instrumentation function %s at "PFX
                ":", func_names[i], pc);
         instrlist_disassemble(dc, pc, ilists[i], dr_get_logfile(dc));
-        pc = instrlist_encode(dc, ilists[i], pc, false);
+        pc = instrlist_encode(dc, ilists[i], pc, true);
```

Comment:
Whoops, you're right. I didn't realize that you couldn't comment on the expanded area.